refactor(kubernetes): Use format strings (#1206)
starship/starship: docs/config/README.md

### Options

| Option                  | Default                                            | Description                                      |
| ----------------------- | -------------------------------------------------- | ------------------------------------------------ |

> | `symbol`                | `"☸ "`                                             | A format string representing the symbol of Rust. |

Reviewer: ```suggestion
| `symbol`                | `"☸ "`                                             | A format string representing the symbol displayed before the Cluster. |
```

Author: Fixed! :+1:

| `format`                | `"on [$symbol$context( \\($namespace\\))]($style) "` | The format for the module.                                            |

> | `style`                 | `"bold blue"`                                       | The style for the module.                                             |

Reviewer 2: It seems that the doc is not consistent with the config.

```suggestion
| `style`                 | `"cyan bold"`                                       | The style for the module.                                             |
```
